playground/css-inject/index.html

```diff
@@ -0,0 +1,37 @@
+<script></script>
+<link rel="stylesheet" href="./linked.css" />
+
+<div class="wrapper">
+  <h1>CSS Inject</h1>
+
+  <p class="linked outside">&lt;linked&gt;</p>
+
+  <p class="imported outside">&lt;imported&gt;</p>
+
+  <custom-element></custom-element>
+</div>
+
+<script type="module">
+  customElements.define(
+    'custom-element',
+    class CustomElement extends HTMLElement {
+      constructor() {
+        super()
+        console.log('ctor')
+        this.attachShadow({ mode: 'open' })
+      }
+
+      connectedCallback() {
+        console.log('connected')
+
+        this.shadowRoot.innerHTML = `
+<p class="linked inside">&lt;linked&gt;</p>
+
+<p class="imported inside">&lt;imported&gt;</p>
+`
+
+        import('./imported.css')
```

Comment:
I am not sure about this feature to "alternate" the `import` globally. Also it does not seem to be flexible when you have multiple shadow roots.

Wondering why you want to do it this implicit way, instead of appending the CSS manually with `?inline`

Reply:
?inline can't be applied (as far as I'm aware) to styles within Vue SFCs. Which is my use case.

Reply:
@antfu does ?inline works for css imported by 3rd party lib?